hicommonwealth/commonwealth
# Added section to display ended contests

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9754

## Description of Changes
Added a new "Previous contests" section on the contests list

## "How We Fixed It"
N/A

## Test Plan
1. Visit contests page of any community with active and finished, one-off and recurring contests
2. Verify you see active contests in the "Active contests" section and ended contests in the "Previous contests" section
3. For recurring contest, verify you see 
    a. only the latest non-ended, non-canceled contest in the active list
    b. all ended/canceled contests in the previous contests list.

## Deployment Plan
N/A

## Other Considerations
N/A
